### PR TITLE
Add Carcinogens from TDC

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

File: data/Carcinogens/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- id: SMILES
+  type: SMILES
+  description: SMILES
+- id: compound_id
+  type: Other
+  description: random drug id
+  names:
+  - compound
+  - compound id
+  - random id
```
Comment (kjappelbaum) [inline data/Carcinogens/meta.yaml:27]:
```suggestion
```

File: data/Carcinogens/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+license: CC BY 4.0
+links:
+- url: https://doi.org/10.1002/qsar.200860192
+  description: corresponding publication
+- url: https://doi.org/10.1021/ci300367a
+  description: corresponding publication
+num_points: 280
+url: https://tdcommons.ai/single_pred_tasks/tox/#carcinogens
```
Comment (kjappelbaum) [inline data/Carcinogens/meta.yaml:35]:
```suggestion
```

File: data/Carcinogens/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  names:
+  - compound
+  - compound id
+  - random id
+license: CC BY 4.0
+links:
+- url: https://doi.org/10.1002/qsar.200860192
+  description: corresponding publication
+- url: https://doi.org/10.1021/ci300367a
+  description: corresponding publication
```
Comment (kjappelbaum) [inline data/Carcinogens/meta.yaml:33]:
```suggestion
  description: corresponding publication
 - url: https://tdcommons.ai/single_pred_tasks/tox/#carcinogens
   description: original dataset
```

**Review by phalem — COMMENTED** [review]
remove compound id

File: data/carcinogens/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    carcinogenesis, the formation of cancer. This may be due to the ability to damage
+    the genome or to the disruption of cellular metabolic processes.
+targets:
+    - id: carcinogen
+      description: whether it is carcinogenic (1) or not (0).
+      units:
+      type: boolean
+      names:
+          - noun: carcinogen
+          - noun: substance, radionuclide, or radiation that promotes carcinogenesis
```
Comment (kjappelbaum) [inline data/carcinogens/meta.yaml:14]:
```suggestion
          - noun: substance that promotes carcinogenesis
```

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
one suggestion, otherwise lgtm!

File: data/carcinogens/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+          - adjective: carcinogenic
+          - gerund: having the potential to cause cancer
+      uris:
+          - http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C347
```
Comment (kjappelbaum) [inline data/carcinogens/meta.yaml:18]:
```suggestion
```
doesn't resolve for me

**Review by MicPie — APPROVED** [review]
thank you all! 😃 lgtm, will merge

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**